steps/identity/appellant-contact-details/answer.html

```diff
@@ -1,34 +1,32 @@
-
+<div class="govuk-summary-list">
+    <dl class="govuk-summary-list__row">
+        <dt class="govuk-summary-list__key">{{ content.cya.address.question | safe }}</dt>
```

Comment:
why do you need to add safe at all? it should only be needed for embedded html and that should only be done on a case by case basis

Reply:
In case characters are encoded (that was the case in Probate for example).